producer-consumer.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: details 
+title: Producer - Consumer 
+main_title: Producer - Consumer 
```

Comment:
remove `main_title`, and then use `h1` in the markdown to give titles

Reply:
Done.